src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  margin: auto;
+  padding: 10px;
+  width: calendar-width(
+    $days-in-a-row, 
+    $square-side, 
+    $distance-between-squares
+  );
+  display: flex;
+  flex-wrap: wrap;
+  gap: $distance-between-squares;
+
+  &__day {
+    width: $square-side;
+    height: $square-side;
+    background-color: #eee;
+    border: 1px solid black;
+    text-align: center;
+    line-height: $square-side;
+    transition-duration: 0.5s;
```

Comment:
list properties you need to animate as here. also move 0.5s to variable
<img width="392" alt="image" src="https://user-images.githubusercontent.com/53611910/201491589-c7ebd94d-b300-438a-8ec5-e85dd0fdb384.png">